### PR TITLE
Website UI Updates

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
Broad Strokes: 
- Updates dashboard UI
- Updates landing page UI
- Updates "Join Waitlist" / "Waitlist Joined" UI
- Adds FAQs page
- Adds secondary navigation on desktop and mobile menu on smaller viewports
- Updates the extension to allow the creating, storing, and editing of alias labels from the dashboard

### CODE REVIEW

File: privaterelay/views.py [hunk 1]
```diff
@@ -44,6 +44,16 @@ def home(request):
...
 
 
+def faq(request):
+  if (not request.user or request.user.is_anonymous):
+    return render(request, 'faq.html')
+  fxa_account = request.user.socialaccount_set.filter(provider='fxa').first()
+  avatar = fxa_account.extra_data['avatar'] if fxa_account else None
+  return render(request, 'faq.html', {
+    'avatar': avatar
+  })
```
Comment (groovecoder) [inline privaterelay/views.py:54]: Filed https://github.com/mozilla/fx-private-relay/issues/385 to make this easier as a follow-up.

File: static/js/app.js [hunk 2]
```diff
@@ -68,23 +76,72 @@ function copyToClipboardAndShowMessage(e) {
...
+
+  // Enable "Delete Anyway" button once the checkbox has been clicked.
+  const confirmDeleteModalActions = confirmDeleteModal.querySelectorAll("button");
+  const deleteAnywayBtn = confirmDeleteModalActions[1];
+  checkbox.addEventListener("change", (e) => {
+    if (checkbox.checked) {
+      deleteAnywayBtn.disabled = false;
+    } else {
+      deleteAnywayBtn.disabled = true;
+    }
```
Comment (groovecoder) [inline static/js/app.js:137]:
```suggestion
    if (checkbox.checked) {
      deleteAnywayBtn.disabled = false;
      return;
    }
    deleteAnywayBtn.disabled = true;
```

File: static/js/app.js [hunk 3]
```diff
@@ -196,38 +201,82 @@ async function addEmailToWaitlist(e) {
...
+  };
+
+  if (aliasCard.classList.contains("show-card-details")) {
+    resizeAliasDetails();
+    window.addEventListener("resize", resizeAliasDetails);
+  } else {
+    aliasCard.style.paddingBottom = "0";
+    window.removeEventListener("resize", resizeAliasDetails);
+  }
```
Comment (groovecoder) [inline static/js/app.js:220]:
```suggestion
  if (aliasCard.classList.contains("show-card-details")) {
    resizeAliasDetails();
    window.addEventListener("resize", resizeAliasDetails);
    return;
  }
  aliasCard.style.paddingBottom = "0";
  window.removeEventListener("resize", resizeAliasDetails);
```

File: static/js/app.js [hunk 4]
```diff
@@ -257,7 +305,25 @@ function watchForInstalledAddon() {
...
+
+  const win = window;
+  const header = document.querySelector("header");
+
+  const setHeader = (yOffset) => {
+    if (yOffset > 300) {
+      header.classList.add("fix-header");
+    } else {
+      header.classList.remove("fix-header");
+    }
```
Comment (groovecoder) [inline static/js/app.js:317]:
```suggestion
    if (yOffset > 300) {
      header.classList.add("fix-header");
      return;
    }
    header.classList.remove("fix-header");
```

File: extension/js/get_profile_data.js [hunk 5]
```diff
@@ -9,21 +9,88 @@
...
+    aliasLabelInput.dataset.label = storedAliasLabel;
+
+    if (storedAliasLabel !== "") {
+      aliasLabelInput.value = storedAliasLabel;
+      aliasLabelWrapper.classList.add("user-created-label");
+    } else {
+      aliasLabelInput.placeholder = defaultAliasLabelText;
+    }
+
+    const forbiddenCharacters = `{}()=;-<>`;
```
[the remark this answers is not in the document]
Comment (lesleyjanenorton) [reply extension/js/get_profile_data.js:37]: No, this is totally first-pass/placeholding for whichever characters you think we should exclude.

File: privaterelay/templates/includes/landing_ctas.html [hunk 6]
```diff
@@ -1,10 +1,6 @@
...
 {% load socialaccount %}
 
-<div class="hero-sign-up-bg bg-fx-gradient fx-gradient-btn invisible">
-  <a data-ga="send-ga-funnel-pings" data-event-category="Add to Firefox" data-event-label="add-to-fx-btn-{{ page_position }}" class="add-to-fx hero-sign-up blue-button puffy" href="https://addons.mozilla.org/firefox/addon/private-relay/">Add to Firefox</a>
-  <a
-  data-entrypoint="relay-sign-in-{{ page_position }}" data-form_type="button"
-  data-ga="send-ga-funnel-pings" data-event-category="Sign In Buttons" data-event-label="sign-in-btn-{{ page_position }}"
-  class="sign-in-btn blue-button puffy" href="{% provider_login_url 'fxa' process='login' %}">Sign In</a>
+<div class="hero-sign-up-bg bg-fx-gradient fx-gradient-btn">
+  <a data-ga="send-ga-funnel-pings" data-event-category="Add to Firefox" data-event-label="add-to-fx-btn-{{ page_position }}" class="join-waitlist-btn hero-sign-up blue-button puffy" href="{% provider_login_url 'fxa' process='login' %}">Get on the waitlist with a Firefox account</a>
```
[the remark this answers is not in the document]
Comment (lesleyjanenorton) [reply privaterelay/templates/includes/landing_ctas.html:5]: good catch. fixing!

**Pull request merged**